docs/section11/Rule11-8.md

```diff
@@ -0,0 +1,54 @@
+# Service_Water_Heating - Rule 11-8
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-8
+
+**Rule Description:** "One system per building area type shall be modeled in the baseline."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b
+
+**Evaluation Context:** B-RMD each SHW BAT
+**Data Lookup:**   
+**Function Call:** 
+- **get_spaces_associated_with_each_SWH_bat**
+- **get_SHW_equipment_associated_with_each_SWH_bat**  
+
+**Applicability Checks:**
+- check that the SHW type in the P_RMD has SHW loads - **is P_RMD appropriate for the applicability check, or should it be B_RMD?**  
+
+## Applicability Checks:
+- only projects with SHW for the SHW space type in the proposed model are expected to have a SHW system in the baseline model.  If there is no SHW in the proposed model, we assume no SHW system is needed.
+- use the function get_spaces_associated_with_each_SWH_bat to get a list of spaces for each SHW type: `shw_and_spaces_dict = get_spaces_associated_with_each_SWH_bat(P_RMD)`
+- look at each SHW bat: `for shw_bat in shw_and_spaces_dict:`
+  - look at each space: `for space_id in shw_and_spaces_dict[shw_bat]:`
+    - get the space using get_component_by_id: `space = get_component_by_id(P_RMD, space_id)`
+    - look for the ServiceWaterHeatingUse in the space.  If even one space has a ServiceWaterHeatingUse, continue to rule logic: `if len(space.service_water_heating_uses) > 0: CONTINUE TO RULE LOGIC`
+  - if the program reaches this line without going to the rule logic, the project is not applicable for this SHW heating use: `NOT_APPLICABLE`
+
+    ## Rule Logic: 
+  - get a dictionary of all SHW equipment connected to the all SHW space bats in the baseline model: `b_shw_equipment_dict = get_SHW_equipment_associated_with_each_SWH_bat(B_RMD)[shw_bat]`
+  - get a dictionary of all SHW equipment connected to the current SHW space bats in the baseline model: `b_shw_equipment_dict_this_use = b_shw_equipment_dict[shw_bat]
+  - create a variable num_swh_systems_this_use: `num_swh_systems = len(b_shw_equipment_dict_this_use["SHWDistribution"])`
+  - create a variable num_swh_equipment_this_use: `num_swh_equipment_this_use = len(b_shw_equipment_dict_this_use["SHWHeatingEq"])`
+  - create a boolean is_referenced_in_other_bats and set it to false: `is_referenced_in_other_bats = false`
+  
+  - if there is exactly one system, check that the system is not used in any of the other BATs: `if num_swh_systems == 1:`
+    - create a variable shw_dist_id and set it to the id of the shw distribution equipment for this bat: `shw_dist_id = b_shw_equipment_dict_this_use["SHWDistribution"][0]`
```

Comment:
inconsistent use of the words "system" and "equipment" is confusing. please change "equipment" to system when referring to ServiceWaterHeatingDistributionSystem